pyproject.toml

```diff
@@ -28,7 +28,7 @@ classifiers = [
     "Typing :: Typed",
 ]
 dependencies = [
-    "copier>=6",
+    "copier>=9.2",
 ]
```

Comment:
What is the need for this change?

Reply:
Thanks, I should have explained that. All versions before 9.2 would break CI because of Pydantic errors or missing module yaml-include.

Reply:
Thanks for clarifying! :ok_hand: